app/javascript/components/dual-group/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Col, Row } from 'patternfly-react';
+
+const Dualgroup = ({ fields, formOptions }) => (
+  <React.Fragment>
+    <Row>
+      <Col md={6}>{formOptions.renderForm([fields[0]], formOptions)}</Col>
```

Comment:
I would probably use something like leftColumn/rightColumn or column size based on field count so we can have 
1, 2, 3, 4, 6, 8, 10 and 12 columns.